pkg/occlient/occlient.go

```diff
@@ -3234,11 +3232,32 @@ func isSubDir(baseDir, otherDir string) bool {
 	return matches
 }
 
-// generateOwnerReference genertes an ownerReference which can then be set as
+// IsRouteSupported checks if route resource type is present on the cluster
+func (c *Client) IsRouteSupported() (bool, error) {
+	const ClusterVersionGroup = "route.openshift.io"
```

Comment:
can we rename it to something `IsClusterOpenshift`, so that it can be used in other things to detect if cluster is openshift or kubernetes, in comments we can mention like `we are using resources unique to openshift to verify it is a openshift cluster`

Reply:
Actually we were advised to check for the specific resources individually.